src/ui/libs/DatalensChartkit/ChartKit/components/ChartKitTooltip/ChartKitTooltip.tsx

```diff
@@ -53,12 +53,18 @@ const ChartKitTooltipComponent = React.forwardRef<ChartKitTooltipRef | undefined
             ref,
             () => ({
                 checkForTooltipNode(e) {
-                    const node = e.target as HTMLElement | null;
+                    let node = e.target as HTMLElement | null;
 
                     if (!node) {
                         return;
                     }
 
+                    const anchorId = node.dataset['tooltipAnchorId'];
+
+                    if (anchorId) {
+                        node = document.getElementById(anchorId) || node;
```

Comment:
Perhaps it was worth looking for a parent element with a specific attribute(data-tooltip-content)? Then you wouldn't have to add additional attributes to all child nodes